### PR TITLE
Updating redundant text

### PR DESCRIPTION
**Pull request by Jordan-Morrison**

### CODE REVIEW

File: _patterns/opt-in.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: default
 name: Opt-in to give consent
 short-description: Notifying the user of data collection and asking them to consent to it.
 feature-img-path: assets/images/opt-in.png
-feature-img-alt: Privacy policy with with checkbox prompt to agree and submit button.
+feature-img-alt: Privacy policy with with**checkbox prompt to agree and submit button.
```
Comment (s-laugh) [inline _patterns/opt-in.md:6]: The word "with" is still duplicated.

**Review by s-laugh — CHANGES_REQUESTED** [review]
Not sure this actually fixes the issue...

ALSO, more importantly, all commits need to be signed and verified. So you will need to sign the commit you already made (basically creating a new commit that is signed).
https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/about-commit-signature-verification

**Comment by Jordan-Morrison** [comment]
Should be all good now!

**Review by s-laugh — APPROVED** [review]

**Comment by s-laugh** [comment]
Looks like you'll need to sign your original commit as well for GitHub to accept the PR.
https://superuser.com/questions/397149/can-you-gpg-sign-old-commits

**Pull request closed without merging**